sdks/RepoToolset/tools/StrongName.targets

```diff
@@ -51,7 +51,7 @@
   </Choose>
 
   <!-- Full signing is not currently supported on non-Windows -->
-  <PropertyGroup Condition="'$(OS)' != 'Windows_NT'">
+  <PropertyGroup Condition="'$(OS)' != 'Windows_NT' and '$(SignAssembly)' != 'false'">
     <PublicSign>true</PublicSign>
```

Comment:
Why is this group even needed? PublicSign is already set to true based on the SignAssembly property above.

Reply:
It's probably not... I'll fix this in a subsequent PR.